test/session.js

```diff
@@ -1269,14 +1532,14 @@ describe('session()', function(){
       .get('/')
       .expect(shouldNotHaveHeader('Set-Cookie'))
       .expect(200, function(err, res){
-        if (err) return done(err);
```

Comment:
Looks like a bunch of accidental changes here with the formatting; I can fix, but figured I'd comment here in case you had to make a change from the question above.